packages/cashier/src/constants/routes-config.js

```diff
@@ -3,14 +3,14 @@ import { routes } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import {
     Cashier,
-    Deposit,
     Withdrawal,
     PaymentAgent,
     AccountTransfer,
     PaymentAgentTransfer,
     P2PCashier,
     OnRamp,
 } from '../containers';
+import { Deposit } from 'Pages/deposit';
```

Comment:
@Bahar-FS as i noticed in @vinu-deriv and @farrah-deriv PR's, and as i understand we decided to import all pages (deposit, onramp, p2p, pa, account-transfer, withdrawal) from `/pages` folder
so we adding `index.js` file into `/pages` folder and export all pages.
so i believe that this line of code should be:
`import { Deposit } from '../pages';`

Reply:
thanks @george-usynin-binary for mentioning that, I added a fix, please take a look again🙏